i18n/qqq.json

```diff
@@ -14,5 +14,7 @@
 	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "Error returned by the API if the “entity” request parameter specifies an unsupported version.\n\nParameters:\n* $1 is the specified (unsupported) version, $2 the supported versions, and $3 the number of supported versions\n",
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Error returned by the API if the “entity” request parameter specifies qualifiers and/or references. They are not yet supported.",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Error returned by the API if the “entity” request parameter doesn't contain a statement that uses the reconciliation property",
-	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Error returned by the API if the “entity” request parameter if the reconciliation statement's mainsnak is not of type value"
+	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Error returned by the API if the “entity” request parameter if the reconciliation statement's mainsnak is not of type value",
+	"wikibasereconcileedit-reconciliationservice-matched-multiple-items": "Error returned by the API if the reconciliation request matches multiple items.\n\nParameters:\n* $1 – the matched items (comma-separated list)\n* $2 – the number of matched items",
+	"wikibasereconcileedit-minimaliteminput-required-property-value-keys": "Error returned by the API if the minimal item input statements parameter doesn't have property and value keys defined.\n\nParameters:\n $1 - the missing key"
```

Comment:
Maybe change “doesn’t have property and value keys defined”? E.g. “is missing a certain key”.

Also, we can remove the `property-value` from the message key. I think just `…-required-keys` should work. (Or maybe `…-required-statements-keys`, because it’s part of the `statements` array.)

Reply:
Good catch! Done.